### ISSUE TITLE
Question: Compact living...

### ISSUE THREAD

**Issue by Lunarwalker60** [spaam/svtplay-dl#1517]
Is there a possibility for a 'bin only' version of svtplay-dl?
It seems unnecessary to install over 3000 lib-files to get a running app.
And do it all over again when there is an update.

Or... have I missed a crucial point somewhere?

**Comment by spaam** [comment]
yes it need a bunch of packages to be able to run, but i dont think its 3000 libs.  🤔  
'bin only' version for what OS? windows? linux? macos? fbsd? obsd?  what cpu architecture?

**Comment by Lunarwalker60** [comment]
In my case it's 64 bit Windows (7 &/ 10) Intel i5.

My prevoius install:
![image](https://user-images.githubusercontent.com/61702762/208412001-ea1f252b-e7a3-4484-8503-88517a9a976c.png)

Sorry, not 3000, only 2790! :-)

**Comment by spaam** [comment]
"filer" is not libs you know :) 
before for windows i only had one file but then i got a bunch of issues that the file was a virus. just a false positive thing. but it was pretty annoying to get those every now and then. i stopped using that thing and use the method i have now.
there might be another way to fix it but then the files are 80+ mb big .exe file.  it had some issues years ago with some dependencies. it might be fixed now.  i dont know :)
